lmdeploy/pytorch/backends/cuda/attention.py

```diff
@@ -210,9 +351,21 @@ def build(
         sliding_window: int = None,
         logical_softcapping: float = None,
         causal: bool = True,
+        use_flash_mla: bool = False,
         **kwargs,
     ) -> TritonAttentionImpl:
         """build."""
+        if use_flash_mla is True:
```

Comment:
if  any of `sliding_window` or `alibi` or `logical_softcapping` is set, we should fall back to `TritonAttentionImpl`